doc/source/user-guide/configuration.rst

```diff
@@ -0,0 +1,81 @@
+.. _ref_user_guide_configuration:
+
+Basic configuration
+===================
+
+Add the following to your `conf.py` file to use the Ansys Sphinx theme:
+
+.. code-block:: python
+
+   html_theme = "ansys_sphinx_theme"
+
+From ansys sphinx theme you can use the following features:
```

Comment:
```suggestion
The Ansys Sphinx Theme provides these features:
```
Add a bulleted list that links to each child section. This will be helpful as the text in each section is relatively long.

Reply:
Sure, added.